---a/net/minecraft/block/BlockLeavesBase.java
+++b/net/minecraft/block/BlockLeavesBase.java
@@ -1,30 +1,19 @@
 package net.minecraft.block;
 
 import net.minecraft.block.material.Material;
-import net.minecraft.util.BlockPos;
-import net.minecraft.util.EnumFacing;
-import net.minecraft.world.IBlockAccess;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class BlockLeavesBase extends Block
-{
+
+public class BlockLeavesBase extends Block {
+
     protected boolean fancyGraphics;
     private static final String __OBFID = "CL_00000326";
 
-    protected BlockLeavesBase(Material materialIn, boolean fancyGraphics)
-    {
+    protected BlockLeavesBase(Material materialIn, boolean fancyGraphics) {
         super(materialIn);
         this.fancyGraphics = fancyGraphics;
     }
 
-    public boolean isOpaqueCube()
-    {
+    public boolean isOpaqueCube() {
         return false;
     }
 
-    @SideOnly(Side.CLIENT)
-    public boolean shouldSideBeRendered(IBlockAccess worldIn, BlockPos pos, EnumFacing side)
-    {
-        return !this.fancyGraphics && worldIn.getBlockState(pos).getBlock() == this ? false : super.shouldSideBeRendered(worldIn, pos, side);
-    }
-}
+}
